compiler/testData/lineNumber/custom/functionCallWithDefault.kt

```diff
@@ -1,3 +1,4 @@
+// IGNORE_BACKEND: JVM_IR
```

Comment:
Don't add new line!  Use empty  one before '// 2 3 ....'

Reply:
Right, I thought the common approach is to use automatic disabling tests. 

If the test gets enabled automatically, It's going to remove this line anyways, right?

Done.

Reply:
In some cases manual changes are required, especially in test for linenumbers and smap. 
If not removed automatically it will fail again.

Reply:
Noted